.gitignore

```diff
@@ -207,6 +207,9 @@ dmypy.json
 ./data/
 src/python/psic/tests/**/output
 
+## John's stats test data
+src/python/psic/stats/data
+
```

Comment:
@JWeb56 Data should exist in the existing data folder. You can create a sub-folder called `stats` in the `data` folder and move it there

Reply:
According to what exactly?